firefox-ios/Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -456,7 +457,9 @@ private extension TabLocationView {
         readerModeButton.isHidden = shoppingButton.isHidden ? newReaderModeState == .unavailable : true
         // When the user turns on the reader mode we need to hide the trackingProtectionButton (according to 16400),
         // we will hide it once the newReaderModeState == .active
-        self.trackingProtectionButton.isHidden = newReaderModeState == .active
+        if newReaderModeState == .active {
+            self.trackingProtectionButton.isHidden = true
+        }
```

Comment:
Why this change?

Reply:
When you switch to reader mode, the status is first send as ".available" and for a brief moment you will see the lock icon, then status changes to ".active" and it disappears.